W-15112255 RPA Manager release notes 2024-02/2. (#338)
mulesoft/docs-rpa: rpa-home/modules/ROOT/pages/_partials/release-notes/rpa-manager-release-notes.adoc

RPA Manager is the cloud-based control plane for your automation projects.

[[february-28-2024]]
== February 28, 2024

// == What's New
include::partial$release-notes/rn-headings.adoc[tag=whatsNew,leveloffset=+2]

> * RPA Manager now supports automatic RPA Bot updates for bots running version 1.5.0 or later.
> // Cris: on January 31 we additionally wrote "When a new version of RPA Bot is available, RPA Manager shows the *Update* button in the *Bots* view of the *Bot Management* module, which enables you to update installed RPA Bots to the latest version." With this release RPA Manager shows this page.

Reviewer: ```suggestion
* RPA Manager now supports automatic RPA Bot updates for bots running version 1.5.0 or later.
+
When a new version of RPA Bot is available, RPA Manager shows the *Update* button in the *Bots* view of the *Bot Management* module, which enables you to update installed RPA Bots to the latest version.
```

> // Cris: on January 31 we additionally wrote "When a new version of RPA Bot is available, RPA Manager shows the *Update* button in the *Bots* view of the *Bot Management* module, which enables you to update installed RPA Bots to the latest version." With this release RPA Manager shows this page.

Reviewer: ```suggestion
```

// == Known Issues
include::partial$release-notes/rn-headings.adoc[tag=knownIssues,leveloffset=+2]

[%header,cols="3a,3a,1a"]
|===
|Known Issue |Workaround | ID
| A new Anypoint Platform user is not available in dropdown lists. | The new Anypoint Platform user must log in at least once to RPA Manager to appear in the lists of available users. | W-13833054
| A user with recently added RPA Project Manager permissions is not available in the dropdown menu for project manager assignments. | The user with the RPA Project Manager permissions must log out from RPA Manager and then log in again to appear in the list of available project managers. | W-13833036

> | Trying to update an RPA Bot, which already received an update task leads to following failed tasks in the RPA Bot's task management. | The update will be performed successfully with the first update task, the following failed update tasks can be ignored. | W-15129879

Reviewer: ```suggestion
| An RPA Bot shows an error in *Tasks* when a user tries to update the bot while the bot is already executing an update task. | Ignore the error message, the first task updates the bot successfully regardless of the message. | W-15129879
```

RPA Manager is the cloud-based control plane for your automation projects.

[[february-26-2024]]
== February 26, 2024

// == Fixed Issues
include::partial$release-notes/rn-headings.adoc[tag=fixedIssues,leveloffset=+2]

[%header,cols="6a,1a"]
|===
|Issue Resolution |ID

> | A user can now select a process in a *Process Evaluation's* _Process Matrix_ to filter for the selected process. | W-14657931

Reviewer: ```suggestion
| You can now select a process in the process matrix inside the *Process Evaluation* module to filter for the selected process. | W-14657931
```

> | The last assigned team within a credential of the *Credential Pool* can now be deleted. | W-14693509

Reviewer: ```suggestion
| You can now remove all teams from a credential of the *Credential Pool*. | W-14693509
```
